TYPO3.Flow/Classes/TYPO3/Flow/Object/ObjectSerializer.php

```diff
@@ -348,7 +352,7 @@ protected function reconstituteArray($dataArray)
      *
      * @param string $type The collection type (class name) to create
      * @param array $dataArray The data array to reconstitute from
-     * @return \Doctrine\Common\Collections\Collection The reconstituted Collection
+     * @return Collection The reconstituted Collection
```

Comment:
I would actually prefer to make it explicit that we are referring the 3rd party doctrine collection here - either through "use Doctrine\Common\Collections as Doctrine;" or keep the FQCN.